### PR TITLE
8316710: Exclude java/awt/font/Rotate/RotatedTextTest.java

### PR DESCRIPTION
**Pull request by RealCLanger**
Exclude java/awt/font/Rotate/RotatedTextTest.java until [JDK-8219641](https://bugs.openjdk.org/browse/JDK-8219641) is fixed. The error shows in several linux installations.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8316710](https://bugs.openjdk.org/browse/JDK-8316710): Exclude java/awt/font/Rotate/RotatedTextTest.java (**Sub-task** - P4)

### Reviewers
 * [Matthias Baesken](https://openjdk.org/census#mbaesken) (@MBaesken - **Reviewer**) ⚠️ Review applies to [ca1339d9](https://git.openjdk.org/jdk/pull/15880/files/ca1339d9030d11e916ddb20161ad963605c9671f)
 * @M4ximumPizza (no known openjdk.org user name / role) ⚠️ Review applies to [ca1339d9](https://git.openjdk.org/jdk/pull/15880/files/ca1339d9030d11e916ddb20161ad963605c9671f)
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/15880/head:pull/15880` \
`$ git checkout pull/15880`

Update a local copy of the PR: \
`$ git checkout pull/15880` \
`$ git pull https://git.openjdk.org/jdk.git pull/15880/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 15880`

View PR using the GUI difftool: \
`$ git pr show -t 15880`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/15880.diff">https://git.openjdk.org/jdk/pull/15880.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/15880#issuecomment-1730874624)

### CODE REVIEW

**Review by MBaesken — APPROVED** [review]

File: test/jdk/ProblemList.txt [hunk 1]
```diff
@@ -468,6 +468,8 @@ sun/java2d/DirectX/OnScreenRenderingResizeTest/OnScreenRenderingResizeTest.java
 java/awt/Window/8159168/SetShapeTest.java 8274106 macosx-aarch64
 java/awt/image/multiresolution/MultiResolutionJOptionPaneIconTest.java 8274106 macosx-aarch64
 
+java/awt/font/Rotate/RotatedTextTest.java 8219641 linux-all
+
```
Comment (aivanov-jdk) [inline test/jdk/ProblemList.txt:472]: Looks good to me, except for its placement: shouldn't it be placed close to other font tests? Shouldn't the list be sorted?

Do we always add newly excluded tests to the end of the list?

Comment (RealCLanger) [reply test/jdk/ProblemList.txt:472]: You are right, it all seems a bit unordered. If you like, I could place it next to line 221 or 416 where other java/awt/font tests are excluded. Not sure what the rule is but it seems like it's at least not alphabetically ordered.

Comment (aivanov-jdk) [reply test/jdk/ProblemList.txt:472]: I couldn't find any best practices for problem-listing.

Yet keeping tests in the same area closer seems like a good idea to me. Either line is fine with me. Thanks.

I started an internal discussion in clientlibs to see whether there are any conventions.

**Review by M4ximumPizza — APPROVED** [review]

File: test/jdk/ProblemList.txt (hunk 1 above)
Comment (aivanov-jdk) [reply test/jdk/ProblemList.txt:472]: > Yet keeping tests in the same area closer seems like a good idea to me. Either line is fine with me.

Let's go for the latter one, line 416.

> I started an internal discussion in clientlibs to see whether there are any conventions.

It looks there are no strict conventions. There groups for *awt* and *swing*, there are also some subheaders to group tests which fail with the same root cause, like the one above the line you're adding.

At the same time, I still think it's better to group tests… This one is in `awt/font`, let's add close to another one.

Other than that, it looks good to me, I'll approve it.

Comment (RealCLanger) [reply test/jdk/ProblemList.txt]: OK, great. I've moved the exclusion to the proposed line. Thanks for checking.

**Review by aivanov-jdk — APPROVED** [review]

File: test/jdk/ProblemList.txt (hunk 1 above)
Comment (aivanov-jdk) [reply test/jdk/ProblemList.txt]: Thank you, Christoph!

**Comment by RealCLanger** [comment]
Thanks Alexey!

/integrate

**Pull request closed without merging**